cli/ssh.go

```diff
@@ -477,6 +482,12 @@ func (r *RootCmd) ssh() *serpent.Command {
 			Description: "Specifies whether to emit SSH output over stdin/stdout.",
 			Value:       serpent.BoolOf(&stdio),
 		},
+		{
+			Flag:        "ssh-host-prefix",
+			Env:         "CODER_CONFIGSSH_SSH_HOST_PREFIX",
```

Comment:
```suggestion
			Env:         "CODER_SSH_SSH_HOST_PREFIX",
```

It would be preferable to use the commands own prefix, is there a reason this was given the configssh prefix?

Reply:
Agreed, made the change.